prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,17 @@
+var languages = [
+  'html',
+  'css',
+  'javascript',
+  'react',
+  'node',
+  'postgres'
+];
+console.log('languages arrays', languages);
+var firstElement = languages[0];
+console.log('the first element of the array is', firstElement);
+var thirdElement = languages[2];
+console.log('the third element of the array is', thirdElement);
+var length = languages.length;
+console.log('the length of the array is', length);
+var lastElement = languages[5];
+console.log('the last element of the array is', lastElement);
```

Comment:
We can calculate the last index of any array by subtracting 1 from its length. The result can be used to get the last element of the array.

```js
var lastElement = languages[length - 1];
```